helm/stargate/templates/stargate_docsapi_deployment.yaml

```diff
@@ -0,0 +1,51 @@
+{{ if .Values.docsapi.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: stargate-docsapi
+  labels:
+    app: stargate-docsapi
+spec:
+  replicas: {{ .Values.docsapi.replicaCount }}
+  selector:
+    matchLabels:
+      app: stargate-docsapi
+  template:
+    metadata:
+      labels:
+        app: stargate-docsapi
+    spec:
+      containers:
+      - name: docsapi
+        image: "stargateio/docsapi:{{ .Values.image.tag }}"
+        ports:
+        - containerPort: 8180
+        readinessProbe:
+          httpGet:
+            path: /stargate/health/ready
+            port: 8180
+          timeoutSeconds: 20
+          initialDelaySeconds: 80
+        livenessProbe:
+          httpGet:
+            path: /stargate/health/live
+            port: 8180
+          initialDelaySeconds: 80
+          timeoutSeconds: 20
+        resources:
+          limits:
+            cpu: {{ mul 4 (.Values.docsapi.cpuReqMillicores | default 500) }}m
+            memory: {{add 500 (.Values.docsapi.memory | default 512)}}Mi
+          requests:
+            cpu: {{ mul 4 (.Values.docsapi.cpuReqMillicores | default 200) }}m
+            memory: {{add 500 (.Values.docsapi.memory | default 512)}}Mi
+        env:
```

Comment:
would add `QUARKUS_HTTP_PORT` and fix it to the `8180` just in case we change the port in the app, the deployment continues to work.. or simply set it to the `8080` as a standarad

Reply:
Why this port need to be configurable? The appropriate port needs to exposed and the service configuration has to change accordingly? Also that won't align with the documentation of stargate. Thoughts?

Reply:
I am not saying to be configurable for the users of the chart, just to add it as ENV and then you are sure the port is fixed here in the deployment spec, no matter if we change the port in the application.yaml.. Understand what I mean?

Reply:
Override of port number not needed to be in sync with stargate documentation.